configuration/device_types/distech_ecy-s1000/module_config.json

```diff
@@ -1,19 +1,13 @@
 {
-  "device_datasheets_url": "*** Device Datasheets URL ***",
-  "device_description": "*** Device Description ***",
-  "device_image": "*** Device Image URL ***",
+  "device_datasheets_url": "https://docs.distech-controls.com/bundle/ECY-S1000_SP/resource/ECY-S1000_SP.pdf",
```

Comment:
something about the directory hierarchy here is not right. I can't remember if we discussed this, but I'm not comfortable with a "configuration" top level (it's way too generic... configuration of what?), and then the underlying levels don't seem coherent.  Can you maybe put together a google doc that outlines the rough structure you're proposing for the "configuration/" directory and then we can have a discussion there?  (Hard to do that effectively in code review comments).

Reply:
Hi @grafnu I've amended this PR to only include the proposal for a template folder to store configuration templates for reports and modules configuration and a device_types library of device specific configurations. I've put the background thinking about this into [a document as suggested](https://docs.google.com/document/d/1CziZjgm3UFzKdA7Bd54sAQpvgm6Oac5hbgjemS4Qd4A/edit#). Thanks for having a look!